beetle/iroh-bitswap/src/handler.rs

```diff
@@ -181,6 +175,8 @@ impl ConnectionHandler for BitswapHandler {
         self.keep_alive
     }
 
+    // TODO(nathanielc): Remove uses of KeepAlive::Until
```

Comment:
one thing that my last job did that I liked a lot was that any TODO comment had to reference a ticket number describing the work that would address the TODO.  So something like `// TODO(WS1-1234): remove this`.

Eventually we hooked up tooling so that whenever a ticket was closed, if there were TODOs in the codebase referencing that ticket number, a notification would be sent reminding the team that the TODO could now be addressed.

What would you think about enforcing a similar rule in the rust-ceramic codebase?

Reply:
I like it, I have created an issue. Will update the PR with the issue link